core/config/locales/en.yml

```diff
@@ -401,6 +402,8 @@ en:
       receive: Receive
       save: Save
       update: Update
+      split: Split
+      ship: ship
```

Comment:
I was wondering the same thing but this is maintaining consistency with the previously used translation and I think when I checked out it shows up in a case sensitive place on a page.